@config/client.config.js

```diff
@@ -26,7 +26,7 @@ const cssLoaders = [
 ];
 
 // root build directory for the SERVER. Client is in /client/
-const buildDir = path.resolve(__dirname, '..', 'build');
+const buildDir = path.resolve(__dirname, '..', isProduction ? 'build' : 'static');
```

Comment:
What is the reason that we change where server files are placed? The buildDirs is where TS -> JS conversion sits, static files should be in a different place.

Reply:
Without this generated client scripts are inaccessible from `/build/client/` without server aliasing for `/static/client`.

It's only necessary for development purposes. Shouldn't affect production ;)